index.html

```diff
@@ -4,9 +4,90 @@
 </head>
 <body>
-    
+    <div class="sign-in">
+        <div class="sign-text">
+            <p>Already have an Intuit account?</p>
+            <a href="">I forgot my User ID or Password</a>
+        </div>
+        <button><img src="images/lock.png"> <p>Sign in</p></button>
+    </div>
+    <div class="title-form">
+        <div class="title">
+            <img src="images/intuit.png" alt="">
+        </div>
+        <div class="apps">
+            <div class="diferent-apps">
+                <img class="mint" src="images/mint.png" alt="">
+                    <p>Mint</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="quickbooks" src="images/quickbooks.png" alt="">
+                    <p>quickbooks</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="turbo" src="images/turbo.png" alt="">
+                    <p>TurboTax</p>
+            </div>    
+        </div>
+    </div>
+    <div class="form">
+        <form action="">
+            <div class="form-head">
+                <div class="form-head-title">
+                    <h1>One Account.</h1>
+                    <h1>Everything Intuit.</h1>
+                </div>
+                <div class="form-head-links">
+                    <p>Create an acount for all of intuit's products</p>
+                    <p>including Mint. <a href="">Learn More</a></p>
+                </div>
+            </div>
+            <div class="form-actions">
+                <div class="actions">
+                    <label for="email">Email Addres</label>
+                    <input type="email" id="email" placeholder="E-mail">
+                </div>
+                <div class="actions">
+                    <label for="phone-number">Phone Number</label>
+                    <input type="number" id="phone-number" placeholder="Phone number">
+                </div>
+                <div class="actions">
+                    <label for="password">Password</label>
+                    <input class="password" type="password" id="password" placeholder="Password">
+                </div>
+                <div class="actions">
+                    <label for="confirm-password">Confirm Password</label>
+                    <input type="password" id="confirm-password" placeholder="Password">
+                </div>
+                
+                <button class="create-account"><img src="images/lock.png"> <p>Create Account</p></button>
+            </div>
+            <div class="disclaimer">
+                <i>By clicking Create Account, you agree to our
+                    <a href="">Terms of Service</a> and have read and acknowledge our <a href="">Privacy Statement.</a></i>
+            </div>
+            
+        </form>
+        <footer>
+            <p>Invisible reCAPTCHA by Google <a href="">Privacy Policy</a> and <a href="">Terms of Use.</a></p>
+        </footer>
+    </div>
+
 </body>
+
+<footer  class="body-footer">
```

Comment:
The footer section should be part of the page body. Move the footer into the body of your HTML.

Reply:
Solved!